src/styles/blocks/bikes.scss

```diff
@@ -0,0 +1,28 @@
+.bikes {
+  &__photo {
+    height: 228px;
+    width: 100%;
+    object-fit: contain;
+
+    @include onDesktop {
+      height: 264px;
+    }
+
+    &:hover {
+      transition: 1s;
```

Comment:
better move `transition` from `:hover` to main element, because it works if when you hover on photo, but when you take back, `transition` does not work 

also add `transition: [value] [duration]`